server/src/main/java/org/elasticsearch/common/logging/DeprecationLogger.java

```diff
@@ -54,16 +50,20 @@ public static DeprecationLogger getLogger(Class<?> aClass) {
      * the "org.elasticsearch" namespace.
      */
     public static DeprecationLogger getLogger(String name) {
-        return new DeprecationLogger(getDeprecatedLoggerForName(name));
+        return new DeprecationLogger(name);
+    }
+
+    private DeprecationLogger(String parentLoggerName) {
+        this.logger = LogManager.getLogger(getLoggerName(parentLoggerName, "deprecation"));
     }
 
-    private static Logger getDeprecatedLoggerForName(String name) {
+    private static String getLoggerName(String name, String prefix) {
```

Comment:
Is required to accept a prefix ? (won't it always be `deprecation` ? (and shouldn't it always be `deprecation` ?)

Reply:
agree, we can remove it